content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -167,8 +228,9 @@ After you upload the new default policy file to the Agent, navigate to the [**Th
 3. **Define search queries:**
    1. Configure a new CSM Threats rule. A rule can have multiple rule cases combined with Boolean logic, for example `(||, &&)`. You can also set the counter, group by, and roll-up window.
 
-    {{< img src="security/cws/workload_security_rules/define_runtime_expression2.png" alt="Adding a rule to the search queries field" >}}
-   - Enter a query so that a trigger is only generated when a value is met. You can also enter suppression queries in the **Suppression Rules**, so that a trigger is not generated when the specified values are met.
+    {{< img src="security/cws/workload_security_rules/define_runtime_expression2.png" alt="Adding a rule to the search queries field" >}}  
```

Comment:
Looks like this needs to be indented by another space or two to be properly aligned with the rest of step 3a.

Reply:
I messed around with it and this was the best I could do.
I'll take another pass but images in indented lists are oddly formatted.

Reply:
I think the trick here is that the beginning of the shortcode has to be indented so it's in line with the beginning of the text - that is, the first `{` of the shortcode needs to be the same number of characters from the beginning of the line as the `C` in `Configure`. It works for me in a local build if I add two spaces at the beginning of the line.

Reply:
I have noticed some inconsistent behaviour with this in the past, though, so maybe you're seeing something different in your local build?

Reply:
fixed it. thanks